plugin/pkg/scheduler/scheduler.go

```diff
@@ -188,7 +188,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {
```

Comment:
Can you please show more concrete explanation for your fix? With the example that you append on this issue, can you please show the example as following?

1) Example without the fix
2) Example with the fix